p_sets/4/collatz.rb

```diff
@@ -0,0 +1,8 @@
+module Collatz
+  def self.steps(int, count = nil)
+    count ||= 0
+    return count if int == 1
+    count += 1
+    int.even? ? steps(int / 2, count) : steps((3 * int + 1), count)
```

Comment:
Personally, I prefer ruby's
```
if
else
end
``` 
syntax to the ternary operator (`? :`) - I think it's more legible. Not sure what our style guide has to say on that though.